portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java

```diff
@@ -1380,7 +1380,7 @@ public Properties toolHeaderProperties(HttpServletRequest req, String skin, Plac
 		retval.setProperty("sakai.html.head.css", headCss);
 		retval.setProperty("sakai.html.head.lang", rloader.getLocale().getLanguage());
 		retval.setProperty("sakai.html.head.css.base", CSSUtils.getCssToolBaseLink(skin,ToolUtils.isInlineRequest(req)));
-		retval.setProperty("sakai.html.head.css.skin", CSSUtils.getCssToolSkinLink(skin));
+		retval.setProperty("sakai.html.head.css.skin", ToolUtils.isInlineRequest(req) ? "" : CSSUtils.getCssToolSkinLink(skin));
```

Comment:
I considered that but it seems awkward to me to have a method that returns a known value based on a boolean input. If the tool is inlined, just don't call it.